tests/components/influxdb/test_init.py

```diff
@@ -1566,7 +1566,8 @@ async def test_invalid_inputs_error(
     with patch(f"{INFLUX_PATH}.time.sleep") as sleep:
         hass.states.async_set("fake.something", 1)
         await hass.async_block_till_done()
-        hass.data[influxdb.DOMAIN].block_till_done()
+        await hass.async_add_executor_job(hass.data[influxdb.DOMAIN].block_till_done)
```

Comment:
There are 18 more places that needs the executor.

Reply:
🙈 will fix